backend/src/routing/controllers/api/Abitur.ts

```diff
@@ -26,21 +62,59 @@ export default class Abitur {
     }
 
     static GETgetAllExams(req: Request, res: Response): void {
-        if (rejectWhenValidationsFail(req, res)) return;
-
         const sql = `
             SELECT
                 abiturpruefungen.id AS examId,
-                studentID AS studentId,
+                art AS examType,
+                partnerStudentName,
+                updatedPartnerStudentName,
+                referenzfach,
+                updatedReferenzfach,
+                bezugsfach,
+                updatedBezugsfach,
+                examiner,
+                updatedExaminer,
+                thema AS topicArea,
+                updatedThema AS updatedTopicArea,
+                problemQuestion,
+                updatedProblemQuestion,
+                presentationForm,
+                updatedPresentationForm,
+                genehmigt AS approved,
                 nutzer.name AS studentName,
-                partnerStudentId,
-                referenzfachID AS referenzfachId,
-                bezugsfachID AS bezugsfachId,
-                betreuendeLehrkraftID AS responsibleTeacherId,
-                thema AS topic
+                studentID AS studentId
             FROM abiturpruefungen, nutzer
             WHERE studentID IS nutzer.id;
             `;
         getAllResults(sql, [], defaultGetAllCallback(res));
     }
+
+    static GETgetExamData(req: Request, res: Response): void {
+        const sql = `
+            SELECT
+                art AS examType,
+                partnerStudentName,
+                updatedPartnerStudentName,
+                referenzfach,
+                updatedReferenzfach,
+                bezugsfach,
+                updatedBezugsfach,
+                examiner,
+                updatedExaminer,
+                thema AS topicArea,
+                updatedThema AS updatedTopicArea,
+                problemQuestion,
+                updatedProblemQuestion,
+                presentationForm,
+                updatedPresentationForm,
+                genehmigt AS approved,
+                ablehnungsgrund AS rejectionReason
+            FROM abiturpruefungen, nutzer
+            WHERE studentID IS nutzer.id AND studentID = ?;
+            `;
+        getAllResults(sql, [getStudentId()], (a: unknown[], err: Error|null): void => {
```

Comment:
GetAll()[0] ?
Warum nicht einfach das normale get von sqlite3? Ist das gleiche, nur built in und daher vermutlich performanter. Aber ebenfalls kein Änderungsbedarf.

Reply:
wird gemacht